### PR TITLE
Make Message windowOptions available for internal customization

### PR DESCRIPTION
**Pull request by bradbeck**
Mostly so functional tests can do something like the following to control the timings associated with message windows.
```
  Ext.merge(t.controller('Message').windowOptions, {
    slideInDuration: 200,
    slideBackDuration: 200,
    autoCloseDelay: 300,
  });
```

### CODE REVIEW

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Message.js [hunk 1]
```diff
@@ -150,6 +150,14 @@ Ext.define('NX.controller.Message', {
   },
 
   /**
+   * Internal customization of message window options.
+   *
+   * @private
```
Comment (jdillon) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Message.js]: @property {Object}

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Message.js [hunk 2]
```diff
@@ -150,6 +150,14 @@ Ext.define('NX.controller.Message', {
   },
 
   /**
+   * Internal customization of message window options.
```
Comment (jdillon) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Message.js]: Should also say... used for functional-testsuite overrides of options ATM.

**Comment by jdillon** [comment]
+1

**Comment by adreghiciu** [comment]
+1

**Comment by bradbeck** [comment]
http://bamboo.s/browse/NX3-OSSF510-1

There was an FT failure, but it is unrelated to this change.

**Pull request merged**